src/functionsDelete.js

```diff
@@ -84,9 +85,7 @@ module.exports = function (functionsToDelete, projectId, appEngineLocation) {
           })
           .then(() => {
             return cloudfunctions.deleteFunction({
-              projectId: projectId,
-              region: helper.getRegion(name),
-              functionName,
+              functionName: name,
```

Comment:
The Google API style guide defines `myfunc` as the "id" and `projects/my-projc/locations/us-central1/functions/myfunc` as the "name". IMO the only solution here is to be pedantically consistent.

Reply:
This definitely is confusing and ive introduced a few bugs during development because of it. I'll go through and switch everything to id and name to follow the style guide